python/ray/monitor.py

```diff
@@ -113,6 +114,19 @@ def __init__(self, redis_address, redis_port, autoscaling_config):
         else:
             self.autoscaler = None
 
+        # Experimental feature: GCS flushing.
+        self.issue_gcs_flushes = "RAY_USE_NEW_GCS" in os.environ
+        self.gcs_flush_policy = None
+        if self.issue_gcs_flushes:
+            # For now, we take the primary redis server to issue flushes,
+            # because task table entries are stored there under this flag.
+            try:
+                self.redis.execute_command("HEAD.FLUSH 0")
+            except redis.exceptions.ResponseError as e:
+                log.info("Turning off flushing due to exception: {}".format(
+                    str(e)))
```

Comment:
This would be a good place to push an error to the driver I think (`ray.utils.push_error_to_driver`).

Reply:
Seems complicated for a harmless operating mode

Reply:
It can happen when (1) Ray is not compiled from source with the flag RAY_USE_NEW_GCS, (2) BUT somehow at run time RAY_USE_NEW_GCS is provided.  This seems a harmless setup issue, not a bug?

Reply:
It's important to push an error to the driver here. Otherwise the user will spend a lot of time trying to figure out what is going wrong.